### ISSUE TITLE
twitter_search_url

### ISSUE THREAD

**Issue by sybesma** [StanScates/Tweet.js-Mod#11]
Thanks for your great work!

Everything is working but search.

I am not an expert but i managed it by changing:
line: 36 Should be: api.twitter.com

And on line 193:

host: s.twitter_search_url,
                    url: "/1.1/search/tweets.json",
                    parameters: $.extend({}, defaults, {

```
                    q: query,
```

**Comment by StanScates** [comment]
You are absolutely correct, it was this way in v1.3, but I changed it because of all the complaints about searching on multiple user names. Will push a fix shortly

**Comment by djiesamsoe** [comment]
I will wait for its update if it works again, 

Regards

**Comment by koraysels** [comment]
@sybesma even with the changes you described, i cannot get search/tweets.json to display all tweets matching my query

**Comment by robertcrowe-zz** [comment]
I changed lines 190-201 in jquery.tweet.js to this and it seems to work:

```
        } else {
            var query = (s.query || 'from:'+s.username.join(' OR from:'));
            return {
                host: s.twitter_api_url,
                url: "/1.1/search/tweets.json",
                parameters: $.extend({}, defaults, {
                    rpp: s.page,
                    q: query,
                    count: count
                })
            };
        }
```
